apps/src/templates/sectionProgress/sectionProgressLoader.js

```diff
@@ -9,7 +9,8 @@ import {
 } from './sectionProgressRedux';
 import {
   processedLevel,
-  processServerSectionProgress
+  processServerSectionProgress,
+  lessonProgressForSection
```

Comment:
General comment for this file.  loadScript seems like a really important function to understand how all the progress data in redux fits together.  We can do this as a separate change, but I think it would be good to have a header comment for loadScript describing what it does and then more comments throughout the function that describe what's going on in a little more detail (e.g. instead of "Combine and transform the data", maybe describe how the data is being combined and transformed.)

Reply:
💯 this file is carrying around a lot of history.